### ISSUE TITLE
iOS Parity?

### ISSUE THREAD

**Issue by m1ga** [arskinner/Appcelerator-CardIO#14]
I've updated the Android version to TiSDK 6+ and added some methods and changed the scanCard() function. It would be great if someone can update the iOS version

* `scanCard()` without callback (will take parameters in future release)
* Events: `complete` and `error`
* Properties: locale(string), cardIOLogo(bool), paypalLogo(bool)
* update to Card.io 5.4.0

**Comment by yozef** [comment]
Would love to make an iOS pull request however, there are issues running my compiled module with `scanViewController.hideCardIOLogo = YES;` by default, when running in Ti App. It has to do with the module (maybe the manifest), however, this is the error I get within Ti App (when running on Device):
```
[ERROR] :  ** BUILD FAILED **
[ERROR] :  The following build commands failed:
[ERROR] :  	Ld build/Intermediates/doobie.build/Debug-iphoneos/doobie.build/Objects-normal/armv7/doobie normal armv7
[ERROR] :  (1 failure)
```

My manifest:
```
version: 2.0.1
apiversion: 2
architectures: armv7 arm64 i386 x86_64
description: Appcelerator wrapper for the Card.io credit card scanning library. Use the phone's camera to read credit card numbers and expiration dates.
author: Likely Solutions
license: Free
copyright: Copyright (c) 2015 by Likely Solutions

# these should not be edited
name: Card.io
moduleid: com.likelysoft.cardio
guid: c0304131-9611-4fff-9f83-f48bc9fa1714
platform: iphone
minsdk: 5.5.1.GA
```

module.config (wasn't there before, but gave it a try):
```OTHER_LDFLAGS=$(inherited) -framework CardIO```

Obviously added also the `libCardIO.a` to `/iphone/CardIO` folder.

Might need @arskinner to figure out what is the culprit here so we can update & build the module.

**Comment by arskinner** [comment]
This has been addressed with 2.0.4
